test/cases/minimal-raw.sh

```diff
@@ -273,7 +281,10 @@ sudo virt-install  --name="${IMAGE_KEY}-uefi"\
                    --noautoconsole \
                    --wait=-1 \
                    --import \
-                   --noreboot
+                   --noreboot \
```

Comment:
Hi @atodorov. This extra-args does not work with `--import` and report `ERROR    Kernel arguments are only supported with location or kernel installs.` Error.